src/mlpack/tests/ann/loss_functions_test.cpp

```diff
@@ -718,7 +718,7 @@ TEST_CASE("HingeEmbeddingLossTest", "[LossFunctionsTest]")
 
   // Test the Backward function
   module.Backward(input, target, output);
-  REQUIRE(arma::as_scalar(arma::accu(output)) == Approx(6).epsilon(1e-3));
+  REQUIRE(arma::as_scalar(accu(output)) == Approx(6).epsilon(1e-3));
```

Comment:
```suggestion
  REQUIRE(accu(output) == Approx(6).epsilon(1e-3));
```
I don't think an `as_scalar()` is necessary for `accu()`, since it already returns a scalar; maybe I overlooked something?  (I know you didn't write that code, I just saw it while going through, so why not clean it up since we're already changing this line? :smile: )

Reply:
Yes, I would only see these if the compiler throws an error, a lot of these `accu` modifications were a regex that is applied on all files.

Reply:
That's why I read every line of the diff :smile: